src/signals/incident/components/form/RadioInput/InputUsingDispatch.js

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
```

Comment:
Is `InputUsingDispatch` an appropriate name for the component? I would change the main compoent to RadioGroupInput and this one to RadioInput because this is what they are.

Reply:
True. Good point. I'll change it.